src/next.js

```diff
@@ -4,7 +4,19 @@ module.exports = {
   parser: '@typescript-eslint/parser',
   overrides: [
     {
-      files: ['**/__tests__/pages/**/*.test.tsx'],
+      files: ['**/cypress/**/*'],
+      rules: {
+        'jest/expect-expect': 'off',
+        'jest/valid-expect-in-promise': 'off',
+        'jest/valid-expect': 'off',
+        'jest/no-standalone-expect': 'off',
+        'testing-library/await-async-query': 'off',
+        'testing-library/prefer-screen-queries': 'off',
+        'testing-library/await-async-utils': 'off',
```

Comment:
Could you elaborate a bit on why you'd like to turn those off?

Reply:
Those are not applicable for cypress, like for example we use cypress to find some element ```cy.findByText('Some text')```, and linter in that case throwing an error and suggest to use `screen`, same for  jest `expect`

Reply:
I just realised now it's cypress specific. Thanks for explaing